From 76ab6932ec75dc92296b910be455b82d89877290 Mon Sep 17 00:00:00 2001
From: czou <cao.zou@windriver.com>
Date: Thu, 20 Aug 2015 17:12:44 +0800
Subject: [PATCH 16/29] ls037v7dw01: fixed the error compatible name

it is error compatible name, change it, so ls037v7dw01 can be rightly
loaded with omap3-panel-sharp-ls037v7dw01.dtsi.

Signed-off-by: czou <cao.zou@windriver.com>
---
 drivers/video/omap2/displays-new/panel-sharp-ls037v7dw01.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/video/omap2/displays-new/panel-sharp-ls037v7dw01.c b/drivers/video/omap2/displays-new/panel-sharp-ls037v7dw01.c
index 4e6eeeb..66653b5 100644
--- a/drivers/video/omap2/displays-new/panel-sharp-ls037v7dw01.c
+++ b/drivers/video/omap2/displays-new/panel-sharp-ls037v7dw01.c
@@ -397,7 +397,7 @@ static int __exit sharp_ls_remove(struct platform_device *pdev)
 }
 
 static const struct of_device_id sharp_ls_of_match[] = {
-	{ .compatible = "omapdss,sharp,ls037v7dw01", },
+	{ .compatible = "sharp,ls037v7dw01", },
 	{},
 };
 
-- 
1.9.1

